### PR TITLE
fix(docs): adds callout for needed parameters in guides

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
## Proposed Changes

_[Adds callout for needed parameters to be provided in multiple guides.]_

## Linked Issues

_[699](https://github.com/fetchai/docs/issues/699)_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [x] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

File: pages/guides/agent-courses/agents-for-ai.mdx [hunk 1]
```diff
@@ -375,11 +379,15 @@ Let's take a look.
     summaryAgent.run()
     ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agent-courses/agents-for-ai.mdx:382]: I don't think we need to callout this twice in the same file?

File: pages/guides/agent-courses/introductory-course.mdx [hunk 2]
```diff
@@ -539,6 +542,10 @@ Similarly, we need to define a script for `bob` so to create a remote communicat
         agent.run()
     ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agent-courses/introductory-course.mdx]: I don't think we need to callout this twice in the same file?

File: pages/guides/agent-courses/introductory-course.mdx [hunk 3]
```diff
@@ -924,6 +931,10 @@ The overall script would be as follow:
         restaurant.run()
     ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agent-courses/introductory-course.mdx]: I don't think we need to callout this thrice in the same file?

File: pages/guides/agent-courses/introductory-course.mdx [hunk 4]
```diff
@@ -1071,6 +1082,10 @@ The overall script would be:
         user.run()
     ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agent-courses/introductory-course.mdx]: I don't think we need to callout this four times in the same file?

File: pages/guides/agents/advanced/name-service.mdx [hunk 5]
```diff
@@ -224,6 +229,10 @@ if __name__ == "__main__":
     alice.run()
 ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agents/advanced/name-service.mdx]: I don't think we need to callout this twice in the same file?

File: pages/guides/agents/advanced/utilizing-api-to-build-network-of-task-and-subtask.mdx [hunk 6]
```diff
@@ -331,6 +335,10 @@ This subtask is triggered if the user wants to read regional or country specific
     agent.include(generate_news_protocol)
     ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agents/advanced/utilizing-api-to-build-network-of-task-and-subtask.mdx]: I don't think we need to callout this twice in the same file?

File: pages/guides/agents/intermediate/communicating-with-other-agents.mdx [hunk 7]
```diff
@@ -328,6 +332,10 @@ if __name__ == "__main__":
     slaanesh.run()
 ```
 
+    <Callout type="warning" emoji="⚠️">
```
Comment (devjsc) [inline pages/guides/agents/intermediate/communicating-with-other-agents.mdx]: I think you get the idea by now

**Review by devjsc — CHANGES_REQUESTED** [review]

File: pages/guides/agents/intermediate/communicating-with-other-agents.mdx (hunk 7 above)
Comment (FelixNicolaeBucsa) [reply pages/guides/agents/intermediate/communicating-with-other-agents.mdx]: Yes

**Pull request merged**